src/sampling.jl

```diff
@@ -16,7 +16,15 @@ Sample elements or points from geometric `object`
 with `method`. Optionally, specify random number
 generator `rng`.
 """
-sample(object, method) = sample(Random.GLOBAL_RNG, object, method)
+sample(object, method::SamplingMethod) = sample(Random.GLOBAL_RNG, object, method)
+
+"""
+    sampleinds(rng, domain, method)
+
+Sample indices of elements in `domain` with discrete
+sampling `method` and random number generator `rng`.
+"""
+function sampleinds end
```

Comment:
Please keep this in the original section of the code. It is only defined for DiscreteSamplingMethod